config/environments/development.rb

```diff
@@ -45,4 +45,6 @@
 
   # The page to create a ticket is open to the world by default?
   config.ticket_creation_is_open_to_the_world_by_default = true
+
+  config.tenant_default_stylesheet_url = nil
```

Comment:
With this approach you can version control a default stylesheet without having to click everywhere just to setup up everything as you have to do while using Joomla or Wordpress like systems

I much prefer rely on version control to store at the very least default configuration bits.
Up to you